scipy/ndimage/_ni_support.py

```diff
@@ -73,7 +73,8 @@ def _get_output(output, input, shape=None):
         shape = input.shape
     if output is None:
         output = numpy.zeros(shape, dtype=input.dtype.name)
-    elif type(output) in [type(type), type(numpy.zeros((4,)).dtype)]:
+    elif isinstance(output, (type, np.dtype)):
```

Comment:
This module just does an `import numpy`; so the `np.dtype` is causing the test failures.

Reply:
opst, thanks, should have noticed/tested locally, yeah, it imports it as `numpy`. Fair enough.